spec/unit/indirector/rest_spec.rb

```diff
@@ -723,6 +723,49 @@ def save_request(key, instance, options={})
     end
   end
 
+  describe '#handle_response' do
```

Comment:
@joshcooper @ericldelaney I figured it out :-D

Reply:
@melissa awesome!